.github/workflows/ci-cd.yml

```diff
@@ -231,7 +232,7 @@ jobs:
 
     - name: Setup Python ${{ matrix.pyver }}
       id: python-install
-      uses: actions/setup-python@v5
+      uses: quansight-labs/setup-python@v5
```

Comment:
Can you provide some more color about this change?

Reply:
Looks like this: https://github.com/actions/setup-python/pull/973#issuecomment-2495900996

Can we again add a TODO comment to revert this once resolved: https://github.com/actions/setup-python/issues/771.